command/flags.go

```diff
@@ -142,8 +140,13 @@ var adminReloadPolicyFlags = []cli.Flag{
 }
 
 var adminSyncFlags = []cli.Flag{
-	peerFlag,
 	indexerHostFlag,
+	&cli.StringFlag{
+		Name:     "peer",
+		Usage:    "Publisher peer ID",
+		Aliases:  []string{"p"},
```

Comment:
can we have these peer IDs differentiated in their name, e.g. 'pubpeer' vs 'propeer' or some way where we've made it clear if the ID for varying commands is referring to the publisher vs the provider?

Reply:
Done